core/src/test/scala/unit/kafka/server/MetadataRequestTest.scala

```diff
@@ -36,22 +36,22 @@ class MetadataRequestTest extends BaseRequestTest {
 
   @Test
   def testClusterIdWithRequestVersion1() {
-    val v1MetadataResponse = sendMetadataRequest(MetadataRequest.allTopics, 1)
+    val v1MetadataResponse = sendMetadataRequest(MetadataRequest.allTopics(1.toShort))
```

Comment:
ok.  will leave it as toShort here since the typecast is kind of annoying visually